daq/topology.py

```diff
@@ -630,6 +650,13 @@ def _append_device_default_allow(self, rules, target_mac):
             acl = {'rule': subrule}
             self._append_augmented_rule(rules, acl)
 
+    def _ethertype_to_port_acl(self, ether_type, ports):
```

Comment:
can you add eth_type to _make_acl_rule and use that instead?   This function is essentially doing the same work that you're doing here... ?

Reply:
Changed.